### PR TITLE
Update cid-redirects.json

### PR DESCRIPTION
**Pull request by stacykor**
Adding in redirect for Nick that I had trouble with on Weds.

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: cid-redirects.json [hunk 1]
```diff
@@ -259,6 +259,7 @@
   "/05Search/Search-Query-Language/Transaction-Analytics/Transactionize-operator": "/docs/search/search-query-language/transaction-analytics/transactionize-operator",
   "/05Search/Subqueries": "/docs/search/subqueries",
   "/05Search/Time-Compare": "/docs/search/time-compare",
+  "/07Sumo-Logic-Apps": "/docs/integrations/sumo-apps",
```
Comment (kimsauce) [inline cid-redirects.json:262]: /07Sumo-Logic-Apps/26Apps_for_Sumo

**Review by kimsauce — DISMISSED** [review]
link change

File: cid-redirects.json (hunk 1 above)
Comment (stacykor) [reply cid-redirects.json:262]: Sadly not the actual request from Nick, but a nice addition.

Comment (kimsauce) [reply cid-redirects.json]: Can you confirm the request from Nick? 

/07Sumo-Logic-Apps = /docs/integrations

and

/07Sumo-Logic-Apps/26Apps_for_Sumo = /docs/integrations/sumo-apps

**Review by kimsauce — APPROVED** [review]

**Pull request merged**